src/olympia/migrations/1018-drop-l10n-from-ratings-body.sql

```diff
@@ -0,0 +1,14 @@
+/* Add the new column.*/
+ALTER TABLE `reviews`
+    ADD COLUMN `text_body` longtext NULL;
+
+/* Copy across the translated values.  There are some ratings that have multiple
+localizations so we're going to use latest (most recent) localization. */
+UPDATE `reviews`
+    SET `text_body` = (
+        SELECT `localized_string` FROM `translations`
+        WHERE `translations`.`id` = `reviews`.`body`
+        ORDER BY `translations`.`modified` DESC
+        LIMIT 1);
+
+/* Clean-up where old column is dropped, etc, in a subsequent migration. */
```

Comment:
And separate deploy, especially the "take the most recent localization" needs some testing in prod to see that we chose the correct ones.